pkg/kube/inject/testdata/webhook/TestWebhookInject_http_probe_nosidecar_rewrite.patch

```diff
@@ -1,84 +1,76 @@
 [
   {
-    "op": "replace",
-    "path": "/spec/containers/0/readinessProbe",
+    "op": "add",
+    "path": "/metadata/labels",
     "value": {
-      "httpGet": {
-        "path": "/app-health/hello/readyz",
-        "port": 15020
-      }
+      "istio.io/rev": "",
+      "security.istio.io/tlsMode": "istio",
+      "service.istio.io/canonical-name": "",
+      "service.istio.io/canonical-revision": "latest"
     }
   },
   {
-    "op": "replace",
-    "path": "/spec/containers/0/livenessProbe",
+    "op": "add",
+    "path": "/metadata/annotations",
     "value": {
-      "httpGet": {
-        "path": "/app-health/hello/livez",
```

Comment:
where did the probes go?

Reply:
I think this is a bug in the test, not the code. I will send another PR

Reply:
Opened https://github.com/istio/istio/pull/28975. Thanks!!